### PR TITLE
Feature topupwallet

### PR DESCRIPTION
**Pull request by Vivek205**
Topup the channel of a general account wallet.

### CODE REVIEW

File: src/Redux/actionCreators/PaymentActions.js [hunk 1]
```diff
@@ -36,6 +37,23 @@ export const executePayment = paymentExecObj => async () => {
   return await executePaymentAPI(token, paymentExecObj);
 };
 
-export const updatePaypalInProgress = (orderId, paymentId, paypalPaymentId, PayerID) => dispatch => {
-  dispatch({ type: UPDATE_PAYPAL_IN_PROGRESS, payload: { orderId, paymentId, paypalPaymentId, PayerID } });
+const orderDetailsAPI = (token, orderId) => {
+  const apiName = APIEndpoints.ORCHESTRATOR.name;
+  const apiPath = `${APIPaths.ORDER_DETAILS}/${orderId}`;
+  // const queryParams = { order_id: orderId };
```
Comment (chetankothari) [inline src/Redux/actionCreators/PaymentActions.js]: Can be deleted.

File: src/components/ServiceDetails/AboutService/ServiceDemo/index.js [hunk 2]
```diff
@@ -77,21 +79,18 @@ class ServiceDemo extends Component {
...
       groupInfo: { group_id: groupId },
-      wallet,
       fetchWallet,
       startFetchWalletLoader,
       stopLoader,
+      firstTimeFetchWallet,
+      updateFirstTimeFetchWallet,
     } = this.props;
-    if (isEmpty(wallet)) {
+    if (firstTimeFetchWallet) {
```
Comment (chetankothari) [inline src/components/ServiceDetails/AboutService/ServiceDemo/index.js:88]: Do you want to start polling after the API is complete ?

**Review by chetankothari — APPROVED** [review]
LGTM

File: src/components/ServiceDetails/AboutService/ServiceDemo/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/ServiceDetails/AboutService/ServiceDemo/index.js:88]: Yh. we need to poll to get the latest channel balance.

File: src/Redux/actionCreators/PaymentActions.js (hunk 1 above)
Comment (Vivek205) [reply src/Redux/actionCreators/PaymentActions.js]: cleaned it

Comment (Vivek205) [reply src/Redux/actionCreators/PaymentActions.js]: https://github.com/singnet/snet-dapp/pull/404/commits/fa4c00dc9951b199b291663830d71106dd382787

**Pull request merged**